Gemfile

```diff
@@ -47,6 +47,7 @@ end
 group :test, :development do
   gem 'test-unit'
   gem 'rails-perftest'
+  gem 'ruby-prof'
```

Comment:
`ruby-prof 0.17.0` is not fully compatible with `rails-perftest` as the `:call_tree` output format breaks things. For full compatibility use `0.15.9`. Check out this [issue](https://github.com/rails/rails-perftest/issues/38) for more on this.

If you are sure only the `:flat` will be used, then go with `ruby-prof 0.17.0` itself.

Reply:
Yeah, that's what I'm doing RN. Let's see which one works best with JUnit.